travis/compile.sh

```diff
@@ -9,8 +9,9 @@ if [[ "$ENABLE_DEBUG" == 1 ]]; then
 else
 	DEBUG="";
 fi
-./buildconf --force
+./buildconf
 ./configure \
+-C \
```

Comment:
Wouldn't it be better to use the long option **--config-cache** here?
-C is not very descriptive.